instrumentation/spring/spring-webflux/spring-webflux-5.3/library/src/main/java/io/opentelemetry/instrumentation/spring/webflux/v5_3/WebfluxServerHttpAttributesGetter.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.spring.webflux.v5_3;
+
+import io.opentelemetry.instrumentation.api.instrumenter.http.HttpServerAttributesGetter;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import javax.annotation.Nullable;
+import org.springframework.web.reactive.HandlerMapping;
+import org.springframework.web.server.ServerWebExchange;
+import org.springframework.web.util.pattern.PathPattern;
+
+enum WebfluxServerHttpAttributesGetter
+    implements HttpServerAttributesGetter<ServerWebExchange, ServerWebExchange> {
+  INSTANCE;
+
+  @Override
+  public String getMethod(ServerWebExchange request) {
+    return request.getRequest().getMethodValue();
+  }
+
+  @Override
+  public List<String> getRequestHeader(ServerWebExchange request, String name) {
+    return request.getRequest().getHeaders().getOrDefault(name, Collections.emptyList());
+  }
+
+  @Nullable
+  @Override
+  public Integer getStatusCode(
+      ServerWebExchange request, ServerWebExchange response, @Nullable Throwable error) {
+    return response.getResponse().getRawStatusCode();
+  }
+
+  @Override
+  public List<String> getResponseHeader(
+      ServerWebExchange request, ServerWebExchange response, String name) {
+    return response.getResponse().getHeaders().getOrDefault(name, Collections.emptyList());
+  }
+
+  @Nullable
+  @Override
+  public String getFlavor(ServerWebExchange request) {
+    return null;
+  }
+
+  @Nullable
+  @Override
+  public String getTarget(ServerWebExchange request) {
+    String path = request.getRequest().getURI().getPath();
+    String query = request.getRequest().getURI().getQuery();
+    if (path == null && query == null) {
+      return null;
+    }
+    if (query != null) {
+      query = "?" + query;
+    }
+    return Optional.ofNullable(path).orElse("") + Optional.ofNullable(query).orElse("");
+  }
+
+  @Nullable
+  @Override
+  public String getRoute(ServerWebExchange request) {
+    Object bestPatternObj = request.getAttribute(HandlerMapping.BEST_MATCHING_PATTERN_ATTRIBUTE);
+    if (bestPatternObj == null) {
+      return null;
+    }
+    String route;
+    if (bestPatternObj instanceof PathPattern) {
+      route = ((PathPattern) bestPatternObj).getPatternString();
+    } else {
+      route = bestPatternObj.toString();
+    }
+    if (route.equals("/**")) {
+      return null; // 404
```

Comment:
with spring boot `/**` doesn't always mean 404 but rather the default route that serves static resources like images etc.

Reply:
Ah I see. I wasn't aware of that. Would you be happy if I removed the comment `//404` and kept the same behaviour?

Reply:
That would be nice. Ideally we should have consistent handling for `/**` if we are ignoring it here we should also ignore it in other spring instrumentations.

Reply:
Ok, I removed the comment. 

The MVC instrumentation is a bit more sophisticated than this when it comes to getting the route. Looks like the `Filter` has access to the application context so it can retrieve the `DispatcherHandler` bean and therefore query the `HandlerMapping` to see if there's a handler for the given request (and if there isn't returns null for route). As a result it doesn't need to handle `/**` specifically. 

From what I see, there's no easy equivalent way to get the `HandlerMapping`s in Webflux, since it's not a given that the `WebFilter` can access the application context(?). It appears that the current method works fine and is less complex, so unsure if there should be any action here to align the two implementations.